show/vxlan.py

```diff
@@ -68,3 +66,243 @@ def tunnel():
         table.append(r)
 
     click.echo(tabulate(table, header))
+
+@vxlan.command()
+def interface():
+    """Show VXLAN VTEP Information"""
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+
+    # Fetching VTEP keys from config DB
+    click.secho('VTEP Information:\n', bold=True, underline=True)
+    vxlan_table = config_db.get_table('VXLAN_TUNNEL')
+    vxlan_keys = vxlan_table.keys()
+    vtep_sip = '0.0.0.0'
+    if vxlan_keys is not None:
+      for key in natsorted(vxlan_keys):
+          key1 = key.split('|',1)
+          vtepname = key1.pop();
+          if 'src_ip' in vxlan_table[key]:
+            vtep_sip = vxlan_table[key]['src_ip']
+          if vtep_sip is not '0.0.0.0':
+             output = '\tVTEP Name : ' + vtepname + ', SIP  : ' + vxlan_table[key]['src_ip']
+          else:
+             output = '\tVTEP Name : ' + vtepname 
+
+          click.echo(output)
+
+    if vtep_sip is not '0.0.0.0':
+       vxlan_table = config_db.get_table('VXLAN_EVPN_NVO')
+       vxlan_keys = vxlan_table.keys()
+       if vxlan_keys is not None:
+         for key in natsorted(vxlan_keys):
+             key1 = key.split('|',1)
+             vtepname = key1.pop();
+             output = '\tNVO Name  : ' + vtepname + ',  VTEP : ' + vxlan_table[key]['source_vtep']
+             click.echo(output)
+
+       vxlan_keys = config_db.keys('CONFIG_DB', "LOOPBACK_INTERFACE|*")
```

Comment:
Is there any description to show that only loopback interface can be set to vtep

Reply:
https://github.com/Azure/SONiC/blob/8645a6a3dd1467153b45ee4f9a08cbf7fd9183d0/doc/vxlan/EVPN/EVPN_VXLAN_HLD.md#512-show-commands

Apart from the show command output there is no other assumption in SONiC components. The VTEP IP should be reachable. Loopback interface is typically used.